scripts/zones/Grand_Palace_of_HuXzoi/IDs.lua

```diff
@@ -20,26 +20,33 @@ zones[xi.zone.GRAND_PALACE_OF_HUXZOI] =
         MEMBERS_LEVELS_ARE_RESTRICTED = 7023, -- Your party is unable to participate because certain members' levels are restricted.
         CONQUEST_BASE                 = 7082, -- Tallying conquest results...
         DOES_NOT_RESPOND              = 7246, -- The gate does not respond...
+        REQUEST_CONFIRMED             = 7367, -- Security portal access request confirmed.Commencing patrol routine. Stay on alert for intruder interference.
+        PATROL_COMPLETED              = 7368, -- Patrol routine completed. Request transfer of final security portal access duty. Awaiting confirmation.
         DUTY_COMPLETE                 = 7369, -- Transfer of final security portal access duty complete.
+        TRANSFER_OF_ACCESS_DUTY       = 7370, -- Transfer of access duty complete.
+        RETURNING_TO_REGULAR_DUTIES   = 7371, -- Returning to regular duties.
         PORTAL_EAST                   = 7372, -- You hear a portal open to the east...
         PORTAL_WEST                   = 7373, -- You hear a portal open to the west...
         PORTAL_NORTH                  = 7374, -- You hear a portal open to the north...
         TIME_EXCEEDED                 = 7375, -- Patrol routine time restriction exceeded. Patrol aborted.
-        RECOMMENCING_PATROL           = 7376, -- Recommencing patrol.
+        PATROL_SUSPENDED              = 7376, -- Patrol suspended. Awaiting orders.
+        RECOMMENCING_PATROL           = 7377, -- Recommencing patrol.
+        RECENTLY_ACTIVATED            = 7378, -- The alcove has been recently activated...
+        TIME_RESTRICTION              = 7379, -- Time restriction: <number> [minute/minutes] (Earth time)
         HOMEPOINT_SET                 = 7464, -- Home point set!
     },
     quasilumin_text =
     {
-        [16916897] = 7369,
-        [16916898] = 7370,
-        [16916899] = 7371,
-        [16916900] = 7372,
-        [16916901] = 7373,
-        [16916902] = 7374,
-        [16916903] = 7375,
-        [16916904] = 7376,
-        [16916905] = 7377,
-        [16916906] = 7378,
+        [16916897] = 7380,
```

Comment:
Eventually, we'll need/want to replace all of these hardcoded ids with some lookups. But in the in interest of getting COP 'complete' we can ignore it for now

Reply:
They're sequential so we could technically replace it with just one "offset" and calculate it from there. `textID = textOffset + (mob:getID() - quasiluminOffset)`

Reply:
+1 for offset

Reply:
The main fear is: If SE decides to put a JP or exemplar camp in Grand Palace, or does _anything else_ to the entity list, these IDs might shift and all of this might break. So looking up using names or offsets from other mobs is a safer way of handling this.